src/components/UpdateNotification.vue

```diff
@@ -52,6 +68,23 @@ const updateInfo = ref({
 })
 const ignoredUpdateVersions = useStorage<string[]>('cockpit-ignored-update-versions', [])
 
+// Wait for 2 seconds for updates. If no updates are available, show a message and emit close.
+onMounted(() => {
+  setTimeout(() => {
+    if (props.modelValue && updateAvailable.value) {
+      showUpdateDialog.value = true
+    } else {
+      openSnackbar({ message: 'No updates available.', variant: 'success' })
```

Comment:
This notification is wrong. If there's an update available but the modelValue props is false, it will enter here.